api/src/httpd/server.ts

```diff
@@ -14,6 +14,9 @@ import fastifyCors from "@fastify/cors";
 import fastifyJwt from "@fastify/jwt";
 import fastifyRateLimit from "@fastify/rate-limit";
 import * as path from "path";
+import { toHttpError } from "http_errors";
+import ajvErrors from "ajv-errors";
+import { addValidationFormats } from "lib/ajvValidation";
```

Comment:
Could you please make this a relative import? `import { addValidationFormats } from "../lib/ajvValidation";`
Absolute paths don't compile on my machine.